packages/cli/cli/versions.yml

```diff
@@ -11,6 +11,14 @@
   irVersion: 55
   version: 0.54.0-rc0
 
+- changelogEntry:
+    - summary: |
+        Update CLI-side markdown parsing to detect more hrefs and srcs (where src files are automatically uploaded),
+        enabling the <Download src="./path/to/file.pdf">Download me</Download> pattern to work.
+      type: fix
+  irVersion: 55
+  version: 0.53.19
```

Comment:
i dont think our automation is smart enough to release this -- youll need to release an rc after 0.54.0